web-ui/src/main/resources/catalog/components/edit/bounding/BoundingDirective.js

```diff
@@ -171,6 +171,12 @@
               return false;
             };
 
+            function getEnlargedExtent(feature) {
```

Comment:
Map now exactly fits the bounds, but this does not look very nice. This adds a buffer to the feature so the map is zoomed out a little.